### ISSUE TITLE
update_last_login still being called

### ISSUE THREAD

**Issue by ghost** [MSA-Argentina/django-no-last-login#1]
I just tried this app on a Django 2.1.7 installation, but `login()` is still causing `update_last_login()` to be called so it doesn't seem to be working properly. Not sure if there's any more useful info I can provide but let me know if so.

**Comment by fmariluis** [comment]
Indeed, we found the same behavior yesterday. Probably something changed in Django since the first release. I'll be looking into it soon.

**Comment by jeremystretch** [comment]
Not sure when this changed in Django, but it looks like we must now include a UID:

```
>>> user_logged_in.disconnect(update_last_login)
False
>>> user_logged_in.disconnect(update_last_login, dispatch_uid='update_last_login')
True
```

**Comment by gdubicki** [comment]
Thanks @jeremystretch, I applied what you suggested in https://github.com/egnyte/django-no-last-login/commit/b2b201a649139cb0e43db6a14d6e28b85eeb0c16 but it doesn't seem to work with Django 3... Any suggestions?
